integration/js/pages/AppPage.js

```diff
@@ -784,6 +784,9 @@ class AppPage {
         + "var sum = imageData.reduce(getSum);"
         + "return sum;"
   }
+  async clickElement(element){
+    await this.clickElement( element);
```

Comment:
Is this correct?
Also should this live in here? This seems like an ulti function that could apply to any page object.

Reply:
This is wrong, I missed a couple cases of the previous usage and did a replace all and didn't catch that I replaced this. Created a new util file to contain this function and updated.